### ISSUE TITLE
Snippet command should not be triggered by built-in completion

### ISSUE THREAD

**Issue by ul** [kak-lsp/kak-lsp#282]
I hit this several times already when my editing was messed up presumably by a false trigger of snippet functionality.

**Comment by MilanVasko** [comment]
Hello, are there any updates on this?

Based on the title and description, I think I'm experiencing the same issue and it's pretty annoying. To illustrate, here's me going through the autocomplete menu:

![kak_lsp_problem](https://user-images.githubusercontent.com/6698045/88430470-c80b7900-cdf8-11ea-92f6-0e42698c5790.png)

When I reach the match snippet, the code gets messed up and I need to undo a few things.

This is my `kakrc`:

```
source "%val{config}/plugins/plug.kak/rc/plug.kak"

plug "ul/kak-lsp" do %{
	cargo build --release --locked
	cargo install --force --path . # `--path .' is needed by recent versions of cargo
} config %{
	hook global WinSetOption filetype=(rust) %{
		lsp-enable-window
	}
}
```

And my `kak-lsp.toml`:

```
[language.rust]
filetypes = ["rust"]
roots = ["Cargo.toml"]
command = "rust-analyzer"
```

When I run `rust-analyzer --version`, I get `rust-analyzer c9c518e`. I'm using `kak-lsp` compiled from commit e1300dc365d4aa2333b8a7d5bf7153ca4c45a93e.

Honestly, I don't care much for snippets, I'd be grateful even if there was just an option to turn them off.

**Comment by ul** [comment]
Setting `snippet_support` to `false` (https://github.com/ul/kak-lsp/blob/master/kak-lsp.toml#L1) would ask language servers to not send snippets (but they ignore it sometimes) and would make kak-lsp to skip snippets handling.

**Comment by MilanVasko** [comment]
I don't know how I missed that, thanks! This will serve as a fine workaround.

**Comment by topisani** [comment]
Should be solved by #616
